cpp/tests/CMakeLists.txt

```diff
@@ -622,6 +622,7 @@ ConfigureTest(
 
 ConfigureTest(STREAM_HASHING_TEST streams/hash_test.cpp STREAM_MODE testing)
 ConfigureTest(STREAM_COPYING_TEST streams/copying_test.cpp STREAM_MODE testing)
+ConfigureTest(STREAM_GROUPBY_TEST streams/groupby_test.cpp STREAM_MODE testing)
```

Comment:
For context, the `STREAM_MODE testing` here is what triggers running the stream validation tests. It is also why we need a separate TU, otherwise the stream validation preload library would be used for all of the tests in the existing groupby tests (and they would fail without additional source code modifications).

Reply:
Gotcha. Thank you for the clarification, @vyasr.